### PR TITLE
Create explainer for Sell Concert Tickets contract

### LINKED ISSUE #939: Page 2 - How to mint tickets to an event
Goal: A new contract to create NFT Drop. 

Use-case: TicketStub is a form that allows users to buy NFTs representing seats. 

This use-case will show 3 rows of seats.

Row 1: $100 each seat (3 NFT tickets)
Row 2: $ 50 each seat (3 NFT tickets)
Row 3: $ 25 each seat (3 NFT tickets)

Result: 
NFT Ticket should land in the User's Smart Wallet.

Relevant docs:
 - [dapp-agoric-basics PRD](https://docs.google.com/document/d/1wUgp4o9pmiva2qRDq-smYolBQ-mzo0uJhCNuQCt6EUg/edit#heading=h.456n8jqmuwqu) google doc
 - [Docs Revamp](https://miro.com/app/board/uXjVNy24pko=/)  miro board

```[tasklist]
### Tasks
- [ ] https://github.com/Agoric/dapp-agoric-basics/pull/8
- [ ] https://github.com/Agoric/documentation/issues/977
- [ ] Document key take away lessons from this contract
- [ ] Video lesson showing highlighting mint code and how it works
```

### PR DESCRIPTION
**Pull request by kbennett2000**
Closes #939
Explainer text and video walkthrough for the Sell Concert Tickets smart contract

### CODE REVIEW

File: main/guides/getting-started/sell-concert-tickets-contract-explainer.md [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+    tradePrice: AmountMath.make(istBrand, 2n),
+    maxTickets: 5n,
+  }
+};
+```
+
+After the contract is initialized, a new ERTP mint for the "Ticket" asset is created:
+```js
+const ticketMint = await zcf.makeZCFMint('Ticket', AssetKind.COPY_BAG);
+const { brand: ticketBrand } = ticketMint.getIssuerRecord();
```
Comment (dckc) [inline main/guides/getting-started/sell-concert-tickets-contract-explainer.md:25]: this is getting a brand; is the voice-over saying that it's _setting_ the brand?
maybe not critical.

File: main/guides/getting-started/swaparoo-how-to-swap-assets-explainer.md [hunk 2]
```diff
@@ -0,0 +1,5 @@
+# Swaparoo Contract
```
Comment (dckc) [inline main/guides/getting-started/swaparoo-how-to-swap-assets-explainer.md:1]: stray file?

**Review by dckc — CHANGES_REQUESTED** [review]
These are getting better and better!

I think you want to get rid of the stray swap explainer file.

**Comment by kbennett2000** [comment]
Closes #939

File: main/guides/getting-started/sell-concert-tickets-contract-explainer.md [hunk 3]
```diff
@@ -0,0 +1,94 @@
...
+    // price from buyer to proceeds
+    [buyerSeat, proceeds, { Price: totalPrice }],
+    // tickets from inventory to buyer
+    [inventorySeat, buyerSeat, want],
+  ]),
+);
+```
+
+As you're going through this tutorial it may be helpful to watch this video walkthrough:
+<iframe width="560" height="315" src="https://www.youtube.com/embed/Wtq6dwsRdOQ" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
```
Comment (dckc) [inline main/guides/getting-started/sell-concert-tickets-contract-explainer.md:94]: how about a link from the video description back to the docs site?

also: how about collecting these explainers in a playlist?

Comment (kbennett2000) [reply main/guides/getting-started/sell-concert-tickets-contract-explainer.md:94]: > how about a link from the video description back to the docs site?

The videos are unlisted on my personal YouTube site. We can list them publicly on the Agoric channel, but I don't have permission to add them there. In their current form, they can only be seen in the embedded player. So nobody is going to see what's in the description anyway.

**Review by dckc — CHANGES_REQUESTED** [review]
since this is a new page, it should get added to nav.js and/or config.mjs

**Pull request closed without merging**